manifest.konnector

```diff
@@ -16,6 +16,10 @@
     "password": {
       "type": "password"
     },
+    "serviceInstanceUrl": {
```

Comment:
If you use "loginUrl" instead of "serviceInstanceUrl", collect will show this field as "URL d'authentification" or else we will have to update the collect application to show this field properly. 

Here is the list of available fields : https://github.com/cozy/cozy-collect/blob/0358b4b786ef57a68e80415110d71d8e141f052b/src/locales/en.json#L35-L64

Reply:
Thanks @doubleface for the review. I've pushed modification on master.